docs/WinRTAPIContracts.md

```diff
@@ -0,0 +1,76 @@
+# Windows App SDK API Contracts
+
+## Summary
+Customers trust Microsoft that the products we make are sound, and will work.  Windows App SDK (WASDK) has recently released version 1.0.  As WASDK evolvs and more developers use WASDK in their own projects WASDK needs a way to guarantee that the API’s that ship are stable and that API versions are well documented.
+To keep customers’ trust in WASDK, and to make sure the WASDK API’s are
+1.  Binary compatible
+2.  Stable
+3.  Customizable so developers can include only the parts they want
+
+Contracts, and contract versions should be added to WASDK.
+
+## Architecture Decisions
+1. Contract boundaries
+Contracts will be split among the winmd boundaries.  This is a good medium between having 1 contract for everything, and 1 contract per class.
```

Comment:
Currently, `Microsoft.UI.Xaml.winmd` contains the `Microsoft.UI.Xaml.Controls` namespace. However, there's no reason that this always needs to be the case. The level of granularity that is required is: "contracts won't span across feeder repos", aka "at least one contract per feeder repo".

Because functionality is typically grouped by namespace, and API contracts will also follow functionality groups, we should expect a lot of overlap between API contracts and namespaces. In other words, the mapping between API contracts and namespaces won't be prescribed, but will be a common natural occurence.

Reply:
We should all have a conversation about this.  Microsoft.Windows.System contains only EM.  Microsoft.Windows.Power contains only power notifications.

Some winmd's have one namespace have one API.